client_plugin/Makefile

```diff
@@ -153,12 +160,15 @@ $(BIN)/$(VMDKOPS_TEST_MODULE).test: $(VMDKOPS_MODULE_SRC) $(TEST_SRC) $(VMDKOPS_
 	$(GO) test -c -o $@ $(PLUGIN)/$(VMDKOPS_MODULE) -cover
 
 $(BIN)/$(PLUGNAME).test: $(COMMON_SRC) $(VMDK_PLUGIN_SRC) $(TEST_SRC) ./vmdk_plugin/*_test.go
-	$(GO) test -c -o $@ $(PLUGIN)/vmdk_plugin -cover
+	$(GO) test -c -o $@ $(PLUGIN)/vmdk_plugin -cover -tags sanity
 
 $(SHARED_PLUGIN_BIN): $(COMMON_SRC) $(SHARED_PLUGIN_SRC)
 	@-mkdir -p $(BIN) && chmod a+w $(BIN)
 	$(GO) build --ldflags '-extldflags "-static"' -o $(SHARED_PLUGIN_BIN) $(PLUGIN)/shared_plugin
 
+$(BIN)/$(INSTRUMENTED_PLUGIN_BIN): $(COMMON_SRC) $(VMDKOPS_MODULE_SRC) $(E2E_TEST_SRC) ./vmdk_plugin/*_test.go
```

Comment:
Please define "VMDK_PLUGIN_TEST_SRC"=./vmdk_plugin/*_test.go" here and use $(VMDK_PLUGIN_TEST_SRC)".